angular/projects/spark-angular/src/lib/components/sprk-masthead/sprk-masthead-selector/sprk-masthead-selector.component.ts

```diff
@@ -13,8 +13,8 @@ import { ISprkMastheadSelectorChoice } from '../sprk-masthead-selector/sprk-mast
 @Component({
   selector: 'sprk-masthead-selector',
   template: `
-    <div [ngClass]="{ 'sprk-c-MastheadMask': isOpen }">
-      <div class="sprk-o-Box">
+    <div [ngClass]="{ 'sprk-c-MastheadMask': isOpen && isFlush }">
+      <div [ngClass]="{ 'sprk-o-Box': isFlush }">
```

Comment:
`isFlush` was part of the React Masthead Selector but wasn't part of Angular yet.

Reply:
Nice. Thank you for adding this! I feel bad having to mention that we also need tests for this but we do haha. Good catch!